static-files.md

```diff
@@ -0,0 +1,137 @@
+# Static Files
+
+Static files include any `css`, `js`, and image files that are not dynamically generated during runtime. `css` is typically compiled from `less` and minified prior to server runtime. `js` files are collected, combined, and minified prior to server runtime. As of this writing we don't compile our JavaScript from a higher level scripting language. Image files generally stay as-is. The only 'dynamic' images come from file attachments in our database.
+
+Due to their static natures, the primary objective for static files is to make as few requests to them during page load, meaning that our goal is to combine static files into one larger, minified file when possible. An additional goal is to ensure that the browser caches the static files it is served to make subsequent page loads faster.
+
+## Collectstatic
+
+Collectstatic is a Django management command that combs through each app's `static` directory and pulls all the static files together under one location, typically a folder at the root of the project.
```

Comment:
Oh yeah, probably.  But either way the point of collectstatic is to gather all the static files into one directory so it can be served by something other than Django (which is comparatively slow for that sort of thing)